voltaire_bundler/utils/eth_client_utils.py

```
@@ -138,30 +179,35 @@ async def send_rpc_request_to_eth_client_no_retry(
+            return json.loads(resp)
+        except json.decoder.JSONDecodeError:
+            logging.critical("Invalid json response from eth client")
+            raise ValueError("Invalid json response from eth client")
+        except Exception as excp:
+            logging.error(
+                "Call to node rpc failed." +
+                str(traceback.format_exc()) +
+                str(excp)
+            )
+            await asyncio.sleep(1)  # in seconds
+        except:
```

Flake8 (7.3.0)

[error] do not use bare 'except'

(E722)